provider-ui/components/Provider.js

```diff
@@ -77,7 +77,7 @@ export default function Provider() {
 
   useEffect(() => {
     loadProvidersFromServer();
-  }, []);
+  }, [loadProvidersFromServer]);
```

Comment:
@Venki1402 have you ensured that there is no re rendering happening after including this dependency?

Reply:
@Venki1402, does this make sense?